apps/e2e/ag-grid-storybook-e2e/src/e2e/data-entry-grid.component.cy.ts

```diff
@@ -0,0 +1,166 @@
+import { E2eVariations } from '@skyux-sdk/e2e-schematics';
+
+describe('ag-grid-storybook', () => {
```

Comment:
This file includes two tests for data entry grid, each loading multiple grids. It's not possible to consolidate them further because overlays preclude any other action.